### PR TITLE
Add doc/how_to_add_a_new_source_file_to_specfem_2D.txt

### PR DESCRIPTION
**Pull request by bottero**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed do not require a buildbot test. The maintainers can commit these changes.

**Comment by buildbot-princeton** [comment]
You pushed to your local branch while your previous changes are waiting to be committed. We assume that you did not intend for these changes to be part of the currently opened pull-request and ignore those changes. If you want these changes to be committed, please open a new pull-request.

**Note to maintainers:** when you accept the original pull-request, the message from buildbot will say that it **closed** the pull-request, as opposed to the usual message that it **merged** the pull-request. In spite of this, the initial pull-request will be merged after you accept it.

**Comment by buildbot-princeton** [comment]
You pushed to your local branch while your previous changes are waiting to be committed. We assume that you did not intend for these changes to be part of the currently opened pull-request and ignore those changes. If you want these changes to be committed, please open a new pull-request.

**Note to maintainers:** when you accept the original pull-request, the message from buildbot will say that it **closed** the pull-request, as opposed to the usual message that it **merged** the pull-request. In spite of this, the initial pull-request will be merged after you accept it.

**Comment by komatits** [comment]
accept

On 03/13/2015 12:37 PM, buildbot-princeton wrote:
> The files changed do not require a buildbot test. The maintainers can
> commit these changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/246#issuecomment-78931010>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

File: doc/how_to_add_a_new_source_file_to_specfem_2D.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Create your source file, for example my_sexy_subroutine.f90 :
+
+      ! Comment here to explain why your subroutine is sexy
+      subroutine my_sexy_subroutine(nothing_funnier_than_that_int)
+
+        use specfem_par, only: nspec,nglob,ibool,coord
+
+        implicit none
+        include "constants.h"
```
Comment (QuLogic) [inline doc/how_to_add_a_new_source_file_to_specfem_2D.txt]: `specfem_par` includes all constants. There should be no reason to include the header as well.

Comment (komatits) [inline doc/how_to_add_a_new_source_file_to_specfem_2D.txt]: Fixed!


On 03/13/2015 09:01 PM, Elliott Sales de Andrade wrote:
> In doc/how_to_add_a_new_source_file_to_specfem_2D.txt
> <https://github.com/geodynamics/specfem2d/pull/246#discussion_r26418417>:
>
>> @@ -0,0 +1,50 @@
>> +STEPS TO ADD A SOURCE FILE TO SPECFEM2D
>> +=======================================
>> +
>> +1. Create your source file, for example my_sexy_subroutine.f90 :
>> +
>> +      ! Comment here to explain why your subroutine is sexy
>> +      subroutine my_sexy_subroutine(nothing_funnier_than_that_int)
>> +
>> +        use specfem_par, only: nspec,nglob,ibool,coord
>> +
>> +        implicit none
>> +        include "constants.h"
>
> |specfem_par| includes all constants. There should be no reason to
> include the header as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/246/files#r26418417>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request closed without merging**